### ISSUE TITLE
Selfhost migrate problems

### ISSUE THREAD

**Issue by Caskia** [exceptionless/Exceptionless#380]
I want to copy my project's apikey to a new environment, how can I do?

**Comment by Caskia** [comment]
So you mean to change data in elastic search, can you provide a interface in ui to do it?

**Comment by niemyjski** [comment]
Yes, you'd need to change the data in elastic search. We have no plans to ever add it to the ui as it is a security risk by allowing users to define api keys.
